store/sqlstore/group_store.go

```diff
@@ -1144,9 +1144,16 @@ func (s *SqlGroupStore) GetGroups(page, perPage int, opts model.GroupSearchOpts)
 
 	groupsQuery = groupsQuery.
 		From("UserGroups g").
-		Where("g.DeleteAt = 0").
 		OrderBy("g.DisplayName")
 
+	if opts.Since > 0 {
+		groupsQuery = groupsQuery.Where(sq.Gt{
```

Comment:
Just a small relization! We may want to use `sq.GtOrEq` since `sq.Gt` returns all the rows that have a `updateAt` greater than the since but not equal too it!

Reply:
@hahmadia The other two places we have a "since" parameter, POSTS and USERS. They use GT. This was the pattern I was following. It depends on how the "since" value is created and stored. It can be done either way, but I would want a good reason to NOT follow the same pattern as POSTS and USERS.